core/domain/wipeout_service.py

```diff
@@ -339,18 +347,160 @@ def _pseudonymize_models(activity_related_models, pseudonymized_user_id):
                 pseudonymized_user_id)
 
 
-def _delete_user_models(user_id):
-    """Delete the user models for the user with user_id.
+def _pseudonymize_feedback_models(pending_deletion_request):
+    """Pseudonymize the activity models for the user with user_id.
 
     Args:
-        user_id: str. The id of the user to be deleted.
+        pending_deletion_request: PendingDeletionRequest. The pending deletion
+            request object to be saved in the datastore.
     """
-    for model_class in models.Registry.get_storage_model_classes(
-            [models.NAMES.user]):
-        if (model_class.get_deletion_policy() not in
-                [base_models.DELETION_POLICY.KEEP,
-                 base_models.DELETION_POLICY.NOT_APPLICABLE]):
-            model_class.apply_deletion_policy(user_id)
+    user_id = pending_deletion_request.user_id
+
+    feedback_thread_model_class = feedback_models.GeneralFeedbackThreadModel
+    feedback_thread_models = feedback_thread_model_class.query(ndb.OR(
+        feedback_thread_model_class.original_author_id == user_id,
+        feedback_thread_model_class.last_nonempty_message_author_id == user_id
+    )).fetch()
```

Comment:
Please check that even if there are > 100 models, this still retrieves all of them (there is a datastore limit of 100 I believe, which applies automatically to some of the queries we do). Ditto below. Probably worth adding a backend test for these cases.

Reply:
Checked and added the test to `test_multiple_feedbacks_are_pseudonymized`.